docs/index.md

```diff
@@ -1809,21 +1809,22 @@ $ REPORTER=nyan npm test
 
 ## More Information
 
-In addition to chatting with us on [Gitter][Gitter-mocha], for additional information such as using
-spies, mocking, and shared behaviours be sure to check out the [Mocha Wiki][Mocha-wiki] on GitHub.
-For discussions join the [Google Group][Google-mocha]. For a running example of Mocha, view
+In addition to chatting with us on [Gitter][gitter-mocha], for additional information such as using
+spies, mocking, and shared behaviours be sure to check out the [Mocha Wiki][mocha-wiki] on GitHub.
+For discussions join the [Google Group][google-mocha]. For a running example of Mocha, view
 [example/tests.html](example/tests.html). For the JavaScript API, view the [API documentation](api/)
 or the [source](https://github.com/mochajs/mocha/blob/master/lib/mocha.js).
 
-[//]: # (Cross reference section)
-
```

Comment:
This blank line must be put back for portability reasons.
Blank lines before **and** after the comment line are [required for portability](https://stackoverflow.com/questions/4823468/comments-in-markdown/20885980#20885980).

Reply:
https://github.com/prettier/prettier/issues/new/choose

Reply:
```md
<!-- prettier-ignore -->
```